packages/opencensus-exporter-stackdriver/src/stackdriver-monitoring.ts

```diff
@@ -183,6 +193,16 @@ export class StackdriverStatsExporter implements StatsEventListener {
     return buckets.map(bucket => bucket.count);
   }
 
+  /**
+   * Gets metric type
+   * @param name The view name
+   * @param prefix Optional metric prefix
+   */
+  private getMetricType(name: string, prefix: string = this.metricPrefix):
```

Comment:
Suggestion: Make private methods not take in optional arguments. These should be set by the caller (call these methods with `metricPrefix != null ? metricPrefix : this.metricPrefix` or similar)

Reply:
Acknowledged